### PR TITLE
Sign in

### PR DESCRIPTION
**Pull request by Ricoch**
Tasks:
- Added sessions controller to handle user sessions
- Added routes for sessions to use our sessions controller and not the default by devise
- Added tests for sessions>create
- Added apiary blueprint

### CODE REVIEW

File: app/controllers/api/v1/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+module Api
+  module V1
+    class SessionsController < DeviseTokenAuth::SessionsController
+      protect_from_forgery with: :null_session
+      include Api::Concerns::ActAsApiRequest
+
+      def resource_params
+        params.require(:user).permit(:email, :password, :password_confirmation)
```
Comment (JuanfraM) [inline app/controllers/api/v1/sessions_controller.rb]: password confirmation it's not necessary for sign in

File: spec/requests/api/v1/sessions/create_spec.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+describe 'POST api/v1/users/sign_in', type: :request do
+  let(:password) { 'password' }
+  let(:token) do
```
Comment (letiesperon) [inline spec/requests/api/v1/sessions/create_spec.rb]: I don't think this token is necessary. Can you try creating the user without the token and see what happens?

Comment (Ricoch) [reply spec/requests/api/v1/sessions/create_spec.rb]: Sure thing, I agree

File: apiary.apib [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    + Attributes
+        + user (Profile Response, required)
+
+# Data Structures
+
+## Profile Response (object)
++ uid: test@test.com (string) - the provider identifier
++ id: 100 (number)
++ email: test@test.com (string)
++ provider: email, facebook, twitter (enum[string]) - it's way the user logins
```
Comment (letiesperon) [inline apiary.apib]: Delete facebook, twitter

**Review by letiesperon — APPROVED** [review]

**Review by JuanfraM — APPROVED** [review]

**Pull request merged**